client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/Sections/Order.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { ReactElement, useMemo, useState } from 'react';
+import {
+  Autocomplete,
+  Box,
+  InputWithLabelRow,
+  NumericTextInput,
+  NumUtils,
+  Typography,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { DraftRequestLine } from '../hooks';
+
+interface OrderProps {
+  disabled?: boolean;
+  isPacksEnabled?: boolean;
+  draft?: DraftRequestLine | null;
+  update: (patch: Partial<DraftRequestLine>) => void;
+}
+
+export const Order = ({
+  disabled,
+  isPacksEnabled,
+  draft,
+  update,
+}: OrderProps): ReactElement => {
+  const t = useTranslation();
+  const [itemType, setItemType] = useState('units');
+
+  const options = useMemo(() => {
+    if (!isPacksEnabled) {
+      return [{ label: t('label.units'), value: 'units' }];
+    }
+    return [
+      { label: t('label.units'), value: 'units' },
+      { label: t('label.packs'), value: 'packs' },
+    ];
+  }, [isPacksEnabled, t]);
+
+  const defaultOption = options.find(option => option.value === itemType);
+
+  const calculatePackQuantity = (precision: number = 0): number => {
+    return NumUtils.round(
+      (draft?.requestedQuantity ?? 0) / (draft?.defaultPackSize ?? 1),
+      precision
+    );
+  };
+
+  const alternativeQuantityLabel = useMemo((): string => {
+    if (itemType === 'packs') {
+      return t('label.order-count-units', {
+        count: Math.ceil(draft?.requestedQuantity ?? 0),
+      });
+    }
+    return t('label.order-count-packs', { count: calculatePackQuantity() });
```

Comment:
I think should include some amount of precision here, or a partial pack warning... I found this confusing?

https://github.com/user-attachments/assets/c90872a0-c605-4963-be48-dc6fbb7c12ea

Reply:
I've applied more precision into it. Please let me know if we find it sufficient enough or if partial pack warning might be betterr